contracts/mocks/ReentrancyAttack.sol

```diff
@@ -5,7 +5,7 @@ contract ReentrancyAttack {
 
   function callSender(bytes4 data) public {
     // solium-disable-next-line security/no-low-level-calls
-    require(msg.sender.call(data));
+    require(msg.sender.call(abi.encodeWithSelector(data)));
```

Comment:
`call` will enforce a single `bytes` argument, this way it will still work properly.